appliances/dell-ftos.gns3a

```diff
@@ -29,7 +29,7 @@
             "version": "9.8.0",
             "md5sum": "b9b50eda0a73407dc381792ff7975e24",
             "filesize": 108115968,
-            "download_url": "https://www.force10networks.com/csportal20/Software/Downloads.aspx",
+            "download_url": "https://www.force10networks.com/CSPortal20/Software/SSeriesDownloads.aspx",
```

Comment:
Note. This is the correct URL for downloading the OS9 image required for the appliance.